apps/src/MusicController.js

```diff
@@ -171,7 +175,7 @@ MusicController.prototype.preload = function() {
  */
 MusicController.prototype.play = function(trackName) {
   debug('play ' + trackName);
-  if (!this.audioPlayer_) {
+  if (!this.audioPlayer_ || this.muteMusic_) {
```

Comment:
Does this mean that this setting will mute the sound now? (I understand we don't yet have a way for user to actually set this setting, but if we set it manually in the DB or something.)

I think you answered this question at checkin but this was the comment I'd half typed out 😅